src/main/java/lekavar/lma/drinkbeer/items/BeerMugItem.java

```diff
@@ -6,7 +6,7 @@
 import net.minecraft.ChatFormatting;
 import net.minecraft.network.chat.Component;
 import net.minecraft.network.chat.Style;
-import net.minecraft.network.chat.TranslatableComponent;
+//import net.minecraft.network.chat.TranslatableComponent;
```

Comment:
Same as for the other file, it may make sense to just remove this commented line.